ebpf/python/gen.go

```diff
@@ -1,4 +1,4 @@
 package python
 
-//go:generate go run github.com/cilium/ebpf/cmd/bpf2go -type global_config_t -type libc -type py_str_type -type py_event -type py_offset_config -target amd64 -cc clang -cflags "-O2 -Wall -Werror -fpie -Wno-unused-variable -Wno-unused-function" Perf ../bpf/pyperf.bpf.c -- -I../bpf/libbpf -I../bpf/vmlinux/
-//go:generate go run github.com/cilium/ebpf/cmd/bpf2go -type global_config_t -type libc -type py_str_type -type py_event -type py_offset_config -target arm64 -cc clang -cflags "-O2 -Wall -Werror -fpie -Wno-unused-variable -Wno-unused-function" Perf ../bpf/pyperf.bpf.c -- -I../bpf/libbpf -I../bpf/vmlinux/
+//go:generate go run github.com/cilium/ebpf/cmd/bpf2go -type libc -type py_str_type -type py_event -type py_offset_config -target amd64 -cc clang-17 -cflags "-O2 -Wall -Werror -fpie -Wno-unused-variable -Wno-unused-function" Perf ../bpf/pyperf.bpf.c -- -I../bpf/libbpf -I../bpf/vmlinux/
```

Comment:
```suggestion
//go:generate go run github.com/cilium/ebpf/cmd/bpf2go -type libc -type py_str_type -type py_event -type py_offset_config -target amd64 -cc clang-17 -cflags "-O2 -Wall -Werror -fpie -Wno-unused-variable -Wno-unused-function" Perf ../bpf/pyperf.bpf.c -- -I../bpf/libbpf -I../bpf/vmlinux/
```

Please ignore, might just be a nixos think.

Not too sure with `clang-17` I get later this `llvm-strip-17 not found`.

```
go generate python/gen.go
Error: exec: "llvm-strip-17": executable file not found in $PATH
```

Both `clang-17` and `clang` are the same.